tests/monitor.cpp

```diff
@@ -92,6 +92,51 @@ TEST_CASE("monitor init event count", "[monitor]")
     CHECK(monitor.total == expected_event_count);
 }
 
+TEST_CASE("monitor init get event count", "[monitor]")
+{
+    common_server_client_setup s{ false };
+    zmq::monitor_t monitor;
+
+    const int expected_event_count = 2;
+    monitor.init(s.client, "inproc://foo");
+
+    int total{ 0 };
+    int connect_delayed{ 0 };
+    int connected{ 0 };
+
+    auto lbd_count_event = [&](const zmq_event_t& event) {
```

Comment:
What does `lbd` stand for?

Reply:
"lbd" is short for "lambda". I like to differentiate "callback" methods over classical variable in my code.

Reply:
Hm. I am not sure if it's necessary to use a prefix at all (if it were a function pointer, would it make a difference?), but if you feel it's good to have, please use `lambda_`.